custom_components/xiaomi_miio/manifest.json

```diff
@@ -3,7 +3,11 @@
   "name": "Xiaomi Nextgen",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/xiaomi_miio",
-  "requirements": ["construct==2.10.68", "micloud==0.6", "git+https://github.com/rytilahti/python-miio.git@master#python-miio==0.6.0"],
+  "requirements": [
+    "construct==2.10.68",
+    "micloud==0.6",
+    "python-miio==0.6.0.dev0"
```

Comment:
I think it'd be best to keep this as it was for the time being, as that will allow development against the master branch w/o dev installation or separate releases until 0.6.0 gets released.

Reply:
Oops; I didn't mean to commit that.

I had added that locally to prevent `hass -c config --skip-pip-packages python-miio` from failing with

```
packaging.requirements.InvalidRequirement: Expected end or semicolon (after name and no valid version specifier)
    git+https://github.com/rytilahti/python-miio.git@master#python-miio==0.6.0
```

Is there another way to run against my local clone of `python-miio`?

Reply:
I have always been using `hass -c config --skip-pip` without issues, but `--skip-pip-packages` would indeed be better (if it worked..).